pl_examples/basic_examples/conf/pl/checkpoint/modelckpt.yaml

```diff
@@ -0,0 +1,12 @@
+# @package _group_
+cls: pytorch_lightning.callbacks.ModelCheckpoint
+params:
+  filepath: None
```

Comment:
what is filepath here?
Where is the model getting saved by default?

Reply:
resolved this was a None value and did not work previously